Kernel_23/doc/Kernel_23/CGAL/intersections.h

```diff
@@ -47,23 +47,15 @@ Also, `Type1` and `Type2` can be both of type
 In three-dimensional space, the types `Type1` and
 `Type2` can be any of the following:
 
+- `Bbox_3`.
 - `Point_3<Kernel>`
 - `Plane_3<Kernel>`
 - `Line_3<Kernel>`
 - `Ray_3<Kernel>`
 - `Segment_3<Kernel>`
+- `Sphere_3<Kernel>`
 - `Triangle_3<Kernel>`.
-- `Bbox_3`.
-
-Also, `Type1` and `Type2` can be respectively of types
-
-- `Triangle_3<Kernel>` and `Tetrahedron_3<Kernel>`
-- `Plane_3<Kernel>` and `Sphere_3<Kernel>` (or the contrary)
-- `Sphere_3<Kernel>` and `Sphere_3<Kernel>`
-- `Line_3<Kernel>` and `Iso_cuboid_3<Kernel>`
-- `Ray_3<Kernel>` and `Iso_cuboid_3<Kernel>`
-- `Segment_3<Kernel>` and `Iso_cuboid_3<Kernel>`
-- `Iso_cuboid_3<Kernel>` and `Iso_cuboid_3<Kernel>`.
+- `Tetrahedron_3<Kernel>`.
```

Comment:
This is wrong (Tetrahedron_3-Tetrahedron_3, Tetrahedron_3-Sphere_3 etc.)

Reply:
We do provide do_intersect(T_3, T_3), and do_intersect(T_3, Sphere_3), etc... The only type that is not accepted is Circle_3, so I don't understand your comment...

Reply:
Misread and thought this was Intersection_3, sorry.